catalog.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Моноподы для селфи</title>
+</head>
+<body>
+  <header class="main-header">
+    <nav class="main-navigation">
+      <a class="header-logo" href="index.html"><img src="https://via.placeholder.com/165x35" width="165" height="35" alt="Логотип интернет-магазина «Девайс»"></a>
+      <form class="search-form" action="https://echo.htmlacademy.ru/" method="GET">
+        <input type="search" name="search" placeholder="Поиск по сайту">
+      </form>
+      <ul class="user-block">
+        <li><a class="profile" href="blank.html">Валентин Петухов</a></li>
+        <li><a class="sign-out" href="blank.html">Выйти</a></li>
+      </ul>
+      <ul class="market-block">
+        <li><a class="compare" href="blank.html">Сравнить</a></li>
+        <li><a class="cart" href="blank.html">Корзина</a></li>
+      </ul>
+      <ul class="main-navigation-list">
+        <li><a href="blank.html">Каталог товаров</a>
+          <ul class="catalog-menu">
+            <li><a href="blank.html">Виртуальная реальность</a></li>
+            <li><a>Моноподы для селфи</a></li>
+            <li><a href="blank.html">Экшн-камеры</a></li>
+            <li><a href="blank.html">Фитнес-браслеты</a></li>
+            <li><a href="blank.html">Умные часы</a></li>
+            <li><a href="blank.html">Квадрокоптеры</a></li>
+          </ul>
+        </li>
+        <li><a href="blank.html">Доставка</a></li>
+        <li><a href="blank.html">Гарантия</a></li>
+        <li><a href="blank.html">Контакты</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <h1>Моноподы для селфи</h1>
+    <ul class="breadcrumbs">
+      <li><a href="index.html">Главная</a></li>
+      <li><a href="blank.html">Каталог товаров</a></li>
+      <li><a>Моноподы для селфи</a></li>
+    </ul>
+    <section class="filters">
+      <h2>Фильтр:</h2>
+      <!--Здесь будет форма фильтра-->
+    </section>
+    <section class="sorting">
+      <h2>Сортировка:</h2>
+      <ul class="sorting-list">
+        <li><a href="blank.html">По цене</a></li>
+        <li><a href="blank.html">По типу</a></li>
+        <li><a href="blank.html">По популярности</a></li>
+        <li><a href="blank.html">По возрастанию</a></li>
+        <li><a href="blank.html">По убыванию</a></li>
+      </ul>
+    </section>
+    <section class="catalog">
+      <h2 class="visually-hidden">Карточки товаров</h2>
+      <ul class="catalog-list">
+        <li class="catalog-list-item">
+          <a href="blank.html"><h3>Любительская селфи-палка</h3></a>
```

Comment:
Лучше сделать наоборот:
```suggestion
          <h3><a href="blank.html">Любительская селфи-палка</a></h3>
```
Заголовок по умолчанию блочный элемент и если его обернуть ссылкой, то кликабельной будет и область, где нет текста.